src/context/ColonyContext.tsx

```diff
@@ -35,21 +38,32 @@ export const ColonyContextProvider = ({
 }) => {
   const { colonyName } = useParams<{ colonyName: string }>();
 
-  const { data, loading, error } = useGetFullColonyByNameQuery({
+  const {
+    data,
+    loading,
+    error,
+    refetch: refetchColony,
+  } = useGetFullColonyByNameQuery({
     skip: !colonyName,
     variables: {
       name: colonyName ?? '',
     },
-    fetchPolicy: 'cache-and-network',
+    fetchPolicy: 'network-only',
+    nextFetchPolicy: 'cache-first',
```

Comment:
`cache-and-network` causes the loading state to reactivate when refetching. It's not obvious as to why, since [the documentation ](https://www.apollographql.com/docs/react/data/queries/#inspecting-loading-states )suggests that this isn't supposed to happen.